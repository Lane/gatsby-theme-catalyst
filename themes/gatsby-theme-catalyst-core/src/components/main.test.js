import React from "react"
import renderer from "react-test-renderer"
import SiteMain from "./main"

describe("Main", () => {

  const props = {
    children: 'test'
  }

  it("renders correctly", () => {
    const tree = renderer
      .create(<SiteMain {...props} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})