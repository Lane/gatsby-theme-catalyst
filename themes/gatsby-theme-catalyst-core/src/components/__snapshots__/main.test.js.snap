// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main renders correctly 1`] = `
<main
  className="css-1spq66s-SiteMain"
>
  test
</main>
`;
